core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -3129,6 +3127,21 @@ object ZIO extends ZIOCompanionPlatformSpecific {
       } *> acc.get
     }
 
+  /**
+   * Merges an `Iterable[IO]` to a single IO, working in with up to `n` fibers in parallel.
+   *
+   * It's unsafe to execute side effects inside `f`, as `f` may be executed
+   * more than once for some of `in` elements during effect execution.
+   */
+  def mergeAllParN[R, E, A, B](n: Int)(
+    in: Iterable[ZIO[R, E, A]]
+  )(zero: B)(f: (B, A) => B): ZIO[R, E, B] =
+    Ref.make(zero) >>= { acc =>
+      foreachParN_(n)(in) {
+        Predef.identity(_) >>= { a => acc.update(f(_, a)) }
```

Comment:
Do you need the `Predef.identity`? 😮

Reply:
It’s shadowed by ZIO.identity 🤷🏼‍♂️

Reply:
Oh sorry I misread. This is silly. Will fix.

I admit to just copying the mergeAllPar implementation 😂